src/modinfo.c

```diff
@@ -2644,6 +2644,28 @@ sr_modinfo_module_data_load_srmon(struct sr_mod_info_s *mod_info, struct sr_mod_
     return err_info;
 }
 
+/**
+ * @brief Check whether the session has pushed any operational data to this module
+ *
+ * @param[in] sess Session to check.
+ * @param[in] mod_name module name to search.
+ *
+ * @return 1 if there is data, else 0
+ * */
```

Comment:
Please keep the doxygen unified, no empty line before `return` and just `*/` at the comment end.